plugins/woocommerce/templates/myaccount/my-address.php

```diff
@@ -60,7 +60,7 @@
 	<div class="u-column<?php echo $col < 0 ? 1 : 2; ?> col-<?php echo $oldcol < 0 ? 1 : 2; ?> woocommerce-Address">
 		<header class="woocommerce-Address-title title">
 			<h3><?php echo esc_html( $address_title ); ?></h3>
-			<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php echo $address ? esc_html__( 'Edit', 'woocommerce' ) : esc_html__( 'Add', 'woocommerce' ); ?></a>
+			<a href="<?php echo esc_url( wc_get_endpoint_url( 'edit-address', $name ) ); ?>" class="edit"><?php echo $address ? esc_html__( 'Edit', 'woocommerce' ) : esc_html__( 'Add', 'woocommerce' ); ?> <?php echo esc_html( $address_title ); ?></a>
```

Comment:
Ideally, we should have used `aria-label` to enhance the accessibility and description, but the text here is hidden using the `:before` pseudo-element in CSS, so that should be fine.

Works well, looks good to me.

![image](https://github.com/woocommerce/woocommerce/assets/15019298/d76fe944-0b94-43ab-a19f-46eb67976de3)